CMakeLists.txt

```diff
@@ -532,8 +532,8 @@ include("${CODING_CONV_CMAKE}/3rdparty.cmake")
 # =============================================================================
 # Add coding-conventions submodule if code formatting enabled
 # =============================================================================
-if(NEURON_CMAKE_FORMAT)
```

Comment:
Seems reasonable to drop this, but there are other references to it in https://github.com/neuronsimulator/nrn/blob/e35bb06faff7719f12187f91f213b82877c84139/CONTRIBUTING.md and https://github.com/neuronsimulator/nrn/blob/e35bb06faff7719f12187f91f213b82877c84139/docs/cmake_doc/options.rst.

Reply:
ah yes, you're right! I forgot